charts/vault-monitoring/values.yaml

```diff
@@ -40,6 +40,7 @@ vault:
     # -- scrapeTimeout of the serviceMonitor
     scrapeTimeout: 30s
     # -- Specify a bearerTokenFile for authentication. Keep empty for Unauthenticated access
+    bearerTokenFile: '/etc/prometheus/config_out/.vault-token'
```

Comment:
do you want the default to be "use a bearer token"? if so, then this is fine, if not, then an `""` might be preferential

Reply:
Yes, as it's highly recommended to secure the connection either by using the bearer token or securing the endpoint with other technologies (like network policies). Also, this way it's not a breaking change.